src/NonceInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace RebelCode\WordPress\Nonce;
+
+use Dhii\Data\IdAwareInterface;
+
+/**
+ * Represents a WordPress nonce.
+ *
+ * A nonce is a disposable code known by the system that is intended to be provided to trusted sources, so that it may
+ * later be sent back to the system, prior to expiration, for authorization and verification purposes.
+ *
+ * @since [*next-version*]
+ */
+interface NonceInterface extends IdAwareInterface
```

Comment:
WordPress calls it an "action", which is incorrect. They call it that because typically they are a slug-like name that represent some operation that needs to take place and that needs the nonce.